src/adams_utils.jl

```diff
@@ -3,7 +3,11 @@
 # III.5 Variable Step Size Multistep Methods: Formulae 5.9
 function ϕ_and_ϕstar!(cache, du, k)
     @inbounds begin
-        @unpack dts, ϕstar_nm1, ϕ_n, ϕstar_n, β = cache
+        @static if VERSION >= v"1.8"
```

Comment:
I wonder if this could be made into a custom `@unpack` macro to avoid the code duplications? Or even just changed in UnPack.jl?

Reply:
Unpack.jl handles cases other than types, while this does not. And this will be easy to delete once LTS changes.